package.json

```diff
@@ -181,6 +181,7 @@
     "@hapi/call": "^8.0.0",
     "@hapi/subtext": "^7.0.3",
     "@kubernetes/client-node": "^0.12.0",
+    "@material-ui/icons": "^4.11.2",
```

Comment:
Is this actually used?

Reply:
Ah this is actually a duplicate of the item in `devDependencies`